selfdrive/debug/uiview.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+import os
+import time
+import signal
+import subprocess
+import multiprocessing
+import cereal.messaging as messaging
+from common.basedir import BASEDIR
+
+
+def send_packets(pm, idx=0):
+  while True:
+    dat = messaging.new_message('controlsState')
+    dat.controlsState.rearViewCam = False
+    pm.send('controlsState', dat)
+    if idx % 50 == 0:  # thermal runs at 2 hz
```

Comment:
For the purposes of this tool, I don't think the freq matters too much, but these are defined in `common.realtime`.

Reply:
Cool, didn't think it did either but wanted to be correct more than anything